doc/man1/s_client.pod

```diff
@@ -514,6 +515,23 @@ specifies the host for the "to" attribute of the stream element.
 If this option is not specified, then the host specified with "-connect"
 will be used.
 
+This option is an alias of the B<-name> option since OpenSSL 1.1.1 for
```

Comment:
Please remove `since OpenSSL 1.1.1`.  If you must, add a remark about this in the `HISTORY` section